src/common/java/com/minelittlepony/gui/GameGui.java

```diff
@@ -0,0 +1,40 @@
+package com.minelittlepony.gui;
+
+import org.apache.commons.lang3.text.WordUtils;
+
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.resources.I18n;
+
+public abstract class GameGui extends GuiScreen {
+
+    @Override
+    protected void actionPerformed(GuiButton button) {
+        if (button instanceof Runnable) {
+            ((Runnable)button).run();
+        }
+    }
+
+    protected static String format(String string, Object... pars) {
+        return I18n.format(string, pars);
+    }
+
+    protected static String toTitleCase(String string) {
+        return WordUtils.capitalize(string.toLowerCase());
```

Comment:
Because it's easier, has a more descriptive name, you don't have to remember to convert the string to lowercase first (something I initially missed) and this way we can change the implementation without visiting all the call sites one by one?